arlas-core/src/main/java/io/arlas/server/model/request/RawGeometries.java

```diff
@@ -19,29 +19,32 @@
 
 package io.arlas.server.model.request;
 
-import io.arlas.server.model.enumerations.AggregatedGeometryStrategyEnum;
+import io.arlas.server.utils.StringUtil;
 
-public class AggregatedGeometry {
-    public AggregatedGeometryStrategyEnum strategy;
-    public String field;
+import java.util.List;
+import java.util.stream.Collectors;
 
-    public AggregatedGeometry() {
+public class RawGeometries {
+    public List<String> geometries;
+    public String sort;
+
+    public RawGeometries() {
     }
 
-    public AggregatedGeometry(AggregatedGeometryStrategyEnum strategy, String field) {
-        this.strategy = strategy;
-        this.field = field;
+    public RawGeometries(List<String> geometries, String sort) {
+        this.geometries = geometries;
+        this.sort = sort;
     }
 
-    public AggregatedGeometry(AggregatedGeometryStrategyEnum strategy) {
-        this.strategy = strategy;
+    public RawGeometries(List<String> geometries) {
+        this.geometries = geometries;
     }
 
     public String flatten() {
-        if (field != null) {
-            return field + '-' + strategy.name();
+        if (!StringUtil.isNullOrEmpty(sort)) {
```

Comment:
Apache StringUtils has a isNotEmpty() and isNotBlank() which are more readable ;)

Reply:
`isNullOrEmpty` is used everywhere in ARLAS-server, I prefer to treat it in another PR